libraries/chain/controller.cpp

```diff
@@ -750,6 +750,24 @@ struct controller_impl {
          // else no checks needed since fork_db will be completely reset on replay anyway
       }
 
+      if (auto dm_logger = get_deep_mind_logger()) {
+         // FIXME: We should probably feed that from CMake directly somehow ...
+         fc_dlog(*dm_logger, "DEEP_MIND_VERSION 12");
```

Comment:
Probably not the best place for this, open to suggestion to where to move it. Used by the reader to determine which format it's about to read.